src/transformers/models/distilbert/modeling_distilbert.py

```diff
@@ -442,6 +442,67 @@ def __init__(self, config):
 
         self.init_weights()
 
+    def get_position_embeddings(self) -> nn.Embedding:
+        """
+        Returns the position embeddings matrix
+        """
+        return self.embeddings.position_embeddings
+
+    def resize_position_embeddings(self, new_num_position_embeddings: int):
+        """
+        Resizes position embeddings matrix of the model if :obj:`new_num_position_embeddings !=
+        config.max_position_embeddings`.
+
+        Arguments:
+            new_num_position_embeddings (:obj:`int`, `optional`):
+                The number of new position embedding matrix. If position embeddings are learned, increasing the size
+                will add newly initialized vectors at the end, whereas reducing the size will remove vectors from the
+                end. If position embeddings not learned (*e.g.* sinusoidal position embeddings), increasing the sise
+                will add correct vectors at the end following the position encoding algorithm, whereas reducing the
+                size will remove vectors from the end.
+        """
```

Comment:
```suggestion
        Arguments:
            new_num_position_embeddings (:obj:`int`, `optional`):
                The number of new position embeddings. If position embeddings are learned, increasing the size
                will add newly initialized vectors at the end, whereas reducing the size will remove vectors from the
                end. If position embeddings are not learned (*e.g.* sinusoidal position embeddings), increasing the size
                will add correct vectors at the end following the position encoding algorithm, whereas reducing the
                size will remove vectors from the end.
        """
```

Reply:
and that would apply to all copies of the doc in other models.